go/vt/vtctl/vtctl.go

```diff
@@ -607,8 +607,19 @@ var commands = []commandGroup{
 					" \nvtctl OnlineDDL test_keyspace show running" +
 					" \nvtctl OnlineDDL test_keyspace show complete" +
 					" \nvtctl OnlineDDL test_keyspace show failed" +
+					" \nvtctl OnlineDDL test_keyspace cleanup 82fa54ac_e83e_11ea_96b7_f875a4d24e90" +
 					" \nvtctl OnlineDDL test_keyspace retry 82fa54ac_e83e_11ea_96b7_f875a4d24e90" +
-					" \nvtctl OnlineDDL test_keyspace cancel 82fa54ac_e83e_11ea_96b7_f875a4d24e90",
+					" \nvtctl OnlineDDL test_keyspace cancel 82fa54ac_e83e_11ea_96b7_f875a4d24e90" +
+					" \nvtctl OnlineDDL test_keyspace cancel-all" +
```

Comment:
IMO `cancel all` is more intuitive, but it's a matter of taste and don't feel strongly about it. The benefit is that it's a single command/signature with the second argument being UUID/`all`.

Reply:
Agreed on `cancel all`. Right now this PR just makes `vtctldclient` compatible with `vtctlclient`. I will later on make `cancel all` available on both.

Reply:
`cancel all` (and likewise `complete all`, `throttle all`, etc.) are now supported. Refactored the code a bit to generalize the parsing of the command.